pom.xml

```diff
@@ -927,19 +927,8 @@
     <profiles>
         <profile>
             <id>ossrh</id>
-            <properties>
```

Comment:
Leaving them in the profile makes the IDE complaining about unset environment variables even though that really doesn't cause any compile/build problems. Its just annoying. So I moved them to settings.xml.